lib/ellipsized.rb

```diff
@@ -41,8 +41,13 @@ class String
   #   "Short".ellipsized # => "Short"
   #   "xyz".ellipsized(0) # => ""
   #   "xyz".ellipsized(2, '...') # => "xy"
+  #
+  # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
@rdavid you can do this in the `.rubocop.yml` file, will look cleaner, I believe